Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -148,19 +133,57 @@ private async Task ReloadComponents()
         {
             var options = _config.GetDlnaConfiguration();
 
-            StartSsdpHandler();
-
             if (options.EnableServer)
             {
+                if (ContentDirectory != null)
```

Comment:
Helper creation only occurs if the server is enabled.
SSDP listening only occurs if the server, or play-to is enabled.